nucleus/payara-modules/notification-core/src/main/java/fish/payara/nucleus/notification/admin/NotificationNotifierConfigurer.java

```diff
@@ -1,202 +0,0 @@
-/*
```

Comment:
Can we not remove commands for the 171 release please?
It would be nicer to just mark it as deprecated, or rip out its `execute` method body and just point it at a newer command (if possible).